fastlane/Fastfile

```diff
@@ -110,6 +113,33 @@ lane :publish_release do |options|
 
   version = options[:version]
 
+  # Update Frameworks (WIP)
+  # update_frameworks
+
+  # Create Github Release
+  github_release = set_github_release(
+                     repository_name: "GetStream/stream-chat-swift",
+                     api_token: ENV["GITHUB_TOKEN"],
+                     name: version,
+                     tag_name: version,
+                     description: changes
+                     # upload_assets: ["Products/StreamChat.zip", "Products/StreamChatUI.zip", "Products/StreamChat-All.zip"]
```

Comment:
These will be added manually _for now_

Reply:
Could we add a comment in `update_frameworks` so that we remember that we should be updating the release there?